velox/exec/Exchange.cpp

```diff
@@ -168,17 +168,18 @@ class LocalExchangeSource : public ExchangeSource {
 std::unique_ptr<ExchangeSource> createLocalExchangeSource(
     const std::string& taskId,
     int destination,
-    std::shared_ptr<ExchangeQueue> queue) {
+    std::shared_ptr<ExchangeQueue> queue,
+    memory::MemoryPool* /* pool */) {
   if (strncmp(taskId.c_str(), "local://", 8) == 0) {
     return std::make_unique<LocalExchangeSource>(
-        taskId, destination, std::move(queue));
+        taskId, destination, std::move(queue), nullptr);
```

Comment:
Why nullptr and not pool?

Reply:
As discussed offline, I will add pool to be passed in, instead of setting a nullptr